lib/EnsEMBL/REST/Model/Variation.pm

```diff
@@ -192,7 +192,7 @@ sub get_gene_phenotype_info {
 
   foreach my $phen (@pfs) {
     my $hash = $self->phen_as_hash($phen);
-    my $key = join("", sort values %$hash);
+    my $key = join("", sort grep {$_} values %$hash);
```

Comment:
grep {$_} means your checking it has a non-0 and non-null value. How would you assume that does nothing unless you don't know what grep does?